# Hi there 👋 I'm Maddie

I'm a computational biologist studying host-microbe interactions in cancer. I love applying my background knowledge in experimental molecular biology to analze omics datasets and uncover new mechanisms that drive disease progression. I am also passionate about utilizing computational methods to discover novel biomarkers for early disease detection. 
<br>

- 🔬 Computational postdoc at the Knight Cancer Instutite within OHSU
- 🥼 Experienced with all things NGS data analysis
- 🧬 Interested in microbiome analysis, transcriptomics, single-cell data, multi-omics, and data integration
- 💻 R, Python, Linux, SLURM
- 🌱 Open to industry career opportunities!
- 📖 Also exploring faculty positions
- 🧑🏻‍🔬 Vice President of [Women in Science PDX](https://www.womeninsciencepdx.org/)

<br>


## 🔗 Connect with me: 
[LinkedIn](https://www.linkedin.com/in/kriegermadeline/)  |   <b>Email:</b> madelineclaire12@gmail.com

<br>

## 🛠 Favorite Tools <br>

Here are some of the tools I regularly use for data analysis, visualization, and reproducible research: <br>
- **Languages**: R, Python, Bash <br>
- **Data Cleaning**: Tidyverse <br>
- **Transcriptomics**: Trimmoatic, DESeq2, Seurat, scRepertoire <br>
- **Microbial Analysis**: Qiime2/DADA2, Phyloseq, Anvi'o <br>
- **Visualization**: ggplot2, Shiny, seaborn, matplotlib <br>
- **Workflows**: Snakemake, R Markdown <br>
- **Environment & Version Control**: Git, GitHub, Conda, SLURM <br>
- **Machine Learning**: LDA topic modeling, scikit-learn, keras <br>

<br>

## 📌 Featured Projects
All the projects below are mostly complete, but some have ongoing components! 🛠️ Please be patient if some scripts are undergoing construction. <br>

1. [Identifying rare tumor-specific T-cell subtypes in circulating peripheral blood](https://github.com/kriegerm/RD_CRC_snakemake)<br>
   **Status**: Ongoing 👷🏻‍♀️ <br>
   **Techniques**: scRNA-seq, TCR-seq  <br>
   **Highlighted Tools**:  <br>
   &nbsp;&nbsp;&nbsp;&nbsp;- Snakemake  
   &nbsp;&nbsp;&nbsp;&nbsp;- Seurat  
   &nbsp;&nbsp;&nbsp;&nbsp;- R markdown  
   &nbsp;&nbsp;&nbsp;&nbsp;- Multi-omic integration  <br>

2. [A novel application of topic modeling to uncover polymicrobial drivers of oral disease in patient-matched samples](https://github.com/kriegerm/PA_cohort_analysis)  <br>
   **Status**: Ongoing 👷🏻‍♀️, mostly complete <br>
   **Techniques**: Microbiome analysis, Differential abundance analysis, Topic modeling  <br>
   **Highlighted Tools**:  <br>
   &nbsp;&nbsp;&nbsp;&nbsp;- Phyloseq  
   &nbsp;&nbsp;&nbsp;&nbsp;- Differential abundance  
   &nbsp;&nbsp;&nbsp;&nbsp;- Latent Dirichlet Allocation topic modeling

3. [Microbiome browser: A shiny app to view and explore microbiome data](http://github.com/kriegerm/Microbiome_browser)  <br>
   **Status**: Complete ✅ <br>
   **Techniques**: Microbiome analysis  <br>
   **Highlighted Tools**:  <br>
   &nbsp;&nbsp;&nbsp;&nbsp;- Phyloseq  
   &nbsp;&nbsp;&nbsp;&nbsp;- R Shiny  <br>

4. [Identifying differential transcriptomic response to subspecies of *Fusobacterium nucleatum* in oral squamous cell carcinoma](https://github.com/kriegerm/Fuso_subsp_OSCC_prelim)  <br>
   **Status**: Complete ✅ <br>
   **Techniques**: Bulk RNA-seq  <br>
   **Highlighted Tools**:  <br>
   &nbsp;&nbsp;&nbsp;&nbsp;- DESeq2   
   &nbsp;&nbsp;&nbsp;&nbsp;- GSEA  
   &nbsp;&nbsp;&nbsp;&nbsp;- R markdown   
   &nbsp;&nbsp;&nbsp;&nbsp;- ggplot2  

<br>

## ⭐️ Other Projects
Many of my projects are in progress, and I am not able to make the code public until a publication is released. I am happy to provide details of my experience in any of these areas upon request!

**Machine Learning**<br>
<i>Project description</i> - I created a <b> Random Forest Classifier </b> using <b> scikit-learn </b> to integrate microbiome and patient lifestyle/diet information in order to determine drivers for cancer risk. Publication coming Fall 2025!

**Data cleaning & large data handling**<br>
<i>Project description</i> - I wrote an automated and reproducible pipeline to clean a dataset containing hundreds of clinical features and written survey answers for downstream use in machine learning applications. Publication coming Fall 2025!

**Creating a novel pangenome for transcriptomic analysis**<br>
<i>Project description</i> - In order to directly compare transcriptomics data from two related bacterial strains, I first created a pangenome of overlapping genes using [Anvi'o](https://anvio.org/). Then, I used that pangenome to map RNA-seq reads from each bacteria in order to generate differential gene expression data for only shared features. Publication coming Winter 2025!

<br>

##  About Me <br>
- 🐴 Horse girl
- 🏃‍♀️ Runner
- ☕️ Coffe enthusiast
- 📖 Reader
- 🐾 Owner of the most perfect dog, Gary!

<br>
<br>

<b><i>Page last updated: </b></i> 03 July 2025
