# Hi there ðŸ‘‹ I'm Maddie

I'm a computational biologist studying host-microbe interactions in cancer. I love applying my background knowledge in experimental molecular biology to analze omics datasets and uncover new mechanisms that drive disease progression. I am also passionate about utilizing computational methods for discovering novel biomarkers for early disease detection. 

- ðŸ”¬ Computational postdoc at the Knight Cancer Instutite within OHSU 
- ðŸ§¬ Interested in microbiome analysis, single-cell data, multi-omics, and data integration
- ðŸ’» R, Python, Linux, SLURM
- ðŸŒ± Open to industry career opportunities!


## ðŸ”— Connect with me: 
[LinkedIn](https://www.linkedin.com/in/kriegermadeline/)

## ðŸ“Œ Featured Projects
All the projects below are mostly complete projects, but some have ongoing components! 

1. [Identifying rare tumor-specific T-cell subtypes in circulating peripheral blood](https://github.com/kriegerm/RD_CRC_snakemake)
   **Status**: Ongoing  
   **Techniques**: scRNA-seq, TCR-seq  
   **Highlighted Tools**:  
   &nbsp;&nbsp;&nbsp;&nbsp;- Snakemake  
   &nbsp;&nbsp;&nbsp;&nbsp;- Seurat  
   &nbsp;&nbsp;&nbsp;&nbsp;- R markdown  
   &nbsp;&nbsp;&nbsp;&nbsp;- Multi-omic integration  

3. [Microbiome browser: A shiny app to view and explore microbiome data](http://github.com/kriegerm/Microbiome_browser)  
   **Status**: Complete  
   **Techniques**: Microbiome analysis  
   **Highlighted Tools**:  
   &nbsp;&nbsp;&nbsp;&nbsp;- Phyloseq  
   &nbsp;&nbsp;&nbsp;&nbsp;- R Shiny  

4. [Identifying differential transcriptomic response to subspecies of *Fusobacterium nucleatum* in oral squamous cell carcinoma](https://github.com/kriegerm/Fuso_subsp_OSCC_prelim)  
   **Status**: Complete  
   **Techniques**: Bulk RNA-seq  
   **Highlighted Tools**:  
   &nbsp;&nbsp;&nbsp;&nbsp;- DESeq2  
   &nbsp;&nbsp;&nbsp;&nbsp;- GSEA  
   &nbsp;&nbsp;&nbsp;&nbsp;- R markdown  
   &nbsp;&nbsp;&nbsp;&nbsp;- ggplot2  

5. [A novel application of topic modeling to uncover polymicrobial drivers of oral disease in patient-matched samples]()  
   **Status**: Ongoing, mostly complete  
   **Techniques**: Microbiome analysis, Differential abundance analysis, Topic modeling  
   **Highlighted Tools**:  
   &nbsp;&nbsp;&nbsp;&nbsp;- Phyloseq  
   &nbsp;&nbsp;&nbsp;&nbsp;- Differential abundance  
   &nbsp;&nbsp;&nbsp;&nbsp;- Latent Dirichlet Allocation topic modeling
   
## Other skills
Many of my projects are in progress, and I am not able to make the code public until a publication is released. I am happy to provide details of my experience in any of these areas upon request!

**Machine Learning**
&nbsp;&nbsp;<i>Project description</i> - I created a <b> Random Forest Classifier </b> using <b> scikit-lear </b> to integrate microbiome and patient lifestyle/diet information in order to determine drivers for cancer risk.

**Data cleaning & large data handling**
&nbsp;&nbsp;<i>Project description</i> - I wrote an automated and reproducible pipeline to clean a dataset containing hundreds of clinical features and written survey answers for downstream use in machine learning applications.
